<?xml version="1.0" encoding="UTF-8"?>
<testrun name="Form1Test" project="ExamplesTest" tests="3" started="3" failures="0" errors="1" ignored="0">
  <testsuite name="com.sencha.extjs.examples.form.dynamic.Form1Test" time="47.201">
    <testcase name="formCanBeCollapsedAndExpanded" classname="com.sencha.extjs.examples.form.dynamic.Form1Test" time="12.638"/>
    <testcase name="formCanBeCollapsedAndExpandedALotOfTimes" classname="com.sencha.extjs.examples.form.dynamic.Form1Test" time="4.258"/>
    <testcase name="weCanTypeOnTextFields" classname="com.sencha.extjs.examples.form.dynamic.Form1Test" time="30.305">
      <error>com.sencha.extjs.driver.ExtDriverException: Expected text [Bukowski de Farias 2], but found [Bukowski de Farias]
	at com.sencha.extjs.driver.Condition.waitUntilSatisfied(Condition.java:19)
	at com.sencha.extjs.driver.AbstractTextFieldFixture.requireText(AbstractTextFieldFixture.java:30)
	at com.sencha.extjs.examples.form.dynamic.Form1Test.weCanTypeOnTextFields(Form1Test.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</error>
    </testcase>
  </testsuite>
</testrun>
